docs/upgrading-fcos.md

```diff
@@ -21,6 +21,115 @@ There are no breaking changes between versions 1.4.0 and 1.5.0 of the `fcos` con
 
 The following is a list of notable new features.
 
+### Offline Tang provisioning
+
+The `tang` section gained a new `advertisement` field. This can be specified in both the `storage` and/or the `boot_device`. If specified, Ignition will use it to provision the Tang server binding rather than fetching the advertisement from the server at runtime. This allows the server to be unavailable at provisioning time. The advertisement can be obtained from the server with `curl http://tang.example.com/adv`.
```

Comment:
Maybe "The `tang` sections in `storage.luks` and `boot_device.luks` gained a new `advertisement` field."?